test/utility/misc.c

```diff
@@ -358,6 +358,42 @@ testutil_print_command_line(int argc, char *const *argv)
     printf("\n");
 }
 
+/*
+ * testutil_wiredtiger_open --
+ *     Call wiredtiger_open with the tiered storage configuration if enabled.
+ */
+void
+testutil_wiredtiger_open(TEST_OPTS *opts, const char *config, WT_EVENT_HANDLER *event_handler,
+  WT_CONNECTION **connectionp, bool recovery)
+{
+    FILE *fp;
+    char buf[512], buf_ext[512], top_level_dir[126];
+
+    strcpy(buf, config);
```

Comment:
I would (almost always) use `strncpy` instead of `strcpy`, and `strncat` instead of `strcat`.  I suspect Coverity would flag this.  `strncpy(buf, config, sizeof(buf));`

Reply:
Replaced `strcpy` with `strncpy` and `strcat` with `strncat`